public/styles.css

```diff
@@ -626,3 +621,61 @@ section#pageThree {
 .projectLabel {
     cursor: pointer;
 }
+@media (min-width: 641px) {
+    #side-navbar {
+        z-index: 11;
+        position: fixed;
+        background-color: #fff;
+        top: 0;
+        right: 0;
+        display: block;
+    }
+}
+@media (max-width: 640px) {
```

Comment:
I think there's room to handle additional layers of screen sizes. For example I can still end up with the following view with ~745px width
![image](https://user-images.githubusercontent.com/22596976/181846605-00e605b7-bd2e-45a2-952e-eb3e514bcff1.png)

Reply:
Awesome!!! Love the design and everything!!

Reply:
I did not finish responsive design for tablets and ultra wide monitors. It takes time, man :)